src/validations/customer-validation.ts

```diff
@@ -0,0 +1,32 @@
+import {Customer} from '../classes/customer';
+import * as customerServices from '../services/customer-service';
+
+const errorList = {
+    customer:`A customer with this account already exists!`,
+}
+
+const checkUser = async (id:number): Promise<boolean> => {
+    if(!!await customerServices.getCustomerByUserId(id))
```

Comment:
!! makes me anxious. Also you could just return the check.
Do a proper check here.